OneSignalSDK/onesignal/in-app-messages/src/main/java/com/onesignal/inAppMessages/internal/InAppMessagesManager.kt

```diff
@@ -208,7 +214,7 @@ internal class InAppMessagesManager(
         }
 
         suspendifyOnThread {
-            fetchMessages()
+            fetchMessages(null)
```

Comment:
We should pass in the offset here. It's a subscription update that we should be waiting on that will probably have a new offset value.